ledger/statefile.go

```diff
@@ -22,6 +23,12 @@ type StateFile struct {
 	HealthcheckUrl string
 }
 
+type ProxyStateFile struct {
```

Comment:
Should this be called `ProxyState` rather than `ProxyStateFile`? I see this is the naming convention - but find it a little confusing. 
Also applies to functions like `loadStateFile` which return State, so could be called `loadState` - the file being internal to the function. (If it was called `loadStateFromFile` I'd expect the file to be a param)